src/query/db_accessor.hpp

```diff
@@ -376,6 +377,20 @@ class DbAccessor final {
     return EdgeAccessor(*maybe_edge);
   }
 
+  storage::Result<EdgeAccessor> ChangeEdgeFrom(EdgeAccessor *edge, VertexAccessor *new_from) {
+    auto changed_edge = static_cast<storage::InMemoryStorage::InMemoryAccessor *>(accessor_)->ChangeEdgeFrom(
```

Comment:
Let's add this function to `src/storage/v2/storage.hpp`` and in place for disk storage throw an error that this is not implemented. This will produce wild bugs if someone with disk storage wants to use it